samples/LoggingTracer/LoggingTracer/LoggingTracer.cs

```diff
@@ -60,33 +60,40 @@ public ISpan StartRootSpan(string operationName, SpanKind kind, SpanCreationOpti
             return new LoggingSpan(operationName, kind);
         }
 
-        /// <inheritdoc/>
-        public ISpan StartSpan(string operationName, SpanKind kind, SpanCreationOptions options)
+        public IDisposable StartActiveRootSpan(string operationName, SpanKind kind, SpanCreationOptions options)
         {
-            Logger.Log($"{this.prefix}.StartSpan({operationName}, {kind}, {options.StartTimestamp:o}, {options.LinksFactory}, {options.Links})");
-            return new LoggingSpan(operationName, kind);
+            return new CurrentSpanUtils.LoggingScope(this.StartRootSpan(operationName, kind, options));
         }
 
-
         /// <inheritdoc/>
         public ISpan StartSpan(string operationName, ISpan parent, SpanKind kind, SpanCreationOptions options)
         {
             Logger.Log($"{this.prefix}.StartSpan({operationName}, {parent.GetType().Name}, {kind}, {options.StartTimestamp:o}, {options.LinksFactory}, {options.Links})");
             return new LoggingSpan(operationName, kind);
         }
 
+        public IDisposable StartActiveSpan(string operationName, ISpan parent, SpanKind kind, SpanCreationOptions options)
+        {
+            return new CurrentSpanUtils.LoggingScope(this.StartSpan(operationName, parent, kind, options));
+        }
+
         /// <inheritdoc/>
         public ISpan StartSpan(string operationName, in SpanContext parent, SpanKind kind, SpanCreationOptions options)
         {
             Logger.Log($"{this.prefix}.StartSpan({operationName}, {parent.GetType().Name}, {kind}, {options.StartTimestamp:o}, {options.LinksFactory}, {options.Links})");
             return new LoggingSpan(operationName, kind);
         }
 
+        public IDisposable StartActiveSpan(string operationName, in SpanContext parent, SpanKind kind, SpanCreationOptions options)
+        {
+            return new CurrentSpanUtils.LoggingScope(this.StartSpan(operationName, parent, kind, options));
+        }
+
         /// <inheritdoc/>
-        public ISpan StartSpanFromActivity(string operationName, Activity activity, SpanKind kind, IEnumerable<Link> links)
+        public IDisposable StartSpanFromActivity(string operationName, Activity activity, SpanKind kind, IEnumerable<Link> links)
```

Comment:
what was wrong with `ISpan`?

Reply:
it's not IDIsposable. Am I talking to the same Sergey in different threads here? :)

Reply:
my question here is why do we need to get a scoped object when start span from activity. Because when we start from regular current Span as a parent - we do not return a scope, only a simple Span

Reply:
I see, so the scenario with Activity is different: we use this method for auto-collectors, when Activity is started before span. Lifetime and scope of the span is controlled by the activity and instrumented library anyway. There is no point in starting span that is not scoped/active for such cases.

Reply:
So this object will stop Activity as well? Perhaps should be a note in `returns` description on what API user must expect